docs/integrations/sources/stripe-migrations.md

```diff
@@ -2,13 +2,35 @@
 
 ## Upgrading to 5.4.0
 
-This change fixes incremental sync issues with the `Refunds` stream:
+The `Refunds` stream previously did not sync incrementally correctly. Incremental syncs are now resolved, and the `Refunds` stream now receives the correct updates using the `events` endpoint. This version resolves incremental sync issues with the `Refunds` stream.
 
-- Stream cursor has changed from `created` to `updated`.
+### Summary of changes: 
 
-The `Reset` for the affected stream `Refunds` is required. It's safe to do, since before this update the `Refunds` stream didn't use the `events` endpoint that have `30 Days data retention` period.
+- The stream `Refunds` cursor changed from the field `created` to `updated`.
+- The stream `Refunds` incrementally syncs using the `events` endpoint.
 
-Because of the changed cursor field of the `Refunds` stream, incremental syncs are now fixed and the stream receives the updates using the `events` endpoint.
+
+### Migration Steps
+
+1. Upgrade the Stripe connector by pressing the upgrade button and following the instructions on the screen.
+
+:::note
+The following migration steps are relevant for those who would like to sync `Refunds` incrementally. 
+:::
+
+The stream `Refunds` will need to be synced historically again to ensure the connection continues syncing smoothly. If available for your destination, we recommend doing a `Refresh` for the stream. 
+
+1. Navigate to the connection's `Schema` tab. Find the `refunds` stream.
+2. Update the `Refunds` stream to use `Incremental | Append + Dedup` sync mode. This ensures your data will sync correctly and capture all updates efficiently. If your stream already has a sync mode of either `Incremental | Append + Dedup` or `Incremental | Append`, simply update the cursor from `created_at` to `updated_at`.
+3. Save the connection.
+4. Review the prompt to `Refresh` the `Refunds` stream. Select `Refresh and retain records` to ensure any data no longer found in Stripe is retained in your destination.
+5. Confirm the modal to save the connection and initiate a `Refresh`. This will start to pull in all historical data for the stream without removing the existing data first and update your destination with all data once complete. 
+
+:::note
+If you are using a destination that does not support the `Refresh` feature, you will need to [Clear](/operator-guides/clear) your stream. This will remove the data from the destination for just that stream. You will then need to sync the connection again in order to sync all data again for that stream.
+:::
+
+The `Refunds` stream previously did not sync using the `events` endpoint, so it retained records beyond 30 days. The `Refunds` stream now uses the `events` endpoint, which limits the results synced to the last 30 days. We recommend retaining a version of your historical data during this process to ensure no data loss occurs.
```

Comment:
I *think* we can still pull refunds for all time when we do a Refresh, and then it just incrementally appends new refunds going forward. Would need someone from the python sources team to confirm.

Reply:
I put this comment here in case people don't have `Refresh` available for their destination, or if they select `Refresh and remove records`. In those cases, they'd lose their historical data I think.

Reply:
I think we may still pull all of the historical data from the `refunds` endpoint (no 30 day limit), and then we sync incremental data from the `events` endpoint (which has the 30 day limit). I am basing this on the Refresh that I ran, which pulled in all historical from 2022. 

So in a `Refresh`, we should sync historical data from the `refunds` endpoint instead of `events`